src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx

```diff
@@ -87,10 +92,10 @@ class TableBodyRow extends Component {
             )}
           </button>
         </Td>
-        {spacers.map(spacerIdx => (
+        {spacers.map((spacerIdx, index) => (
```

Comment:
If `idx` is abbreviated once it should be everywhere, consistently.

Reply:
fixed